### ISSUE TITLE
Missing "@throws" in DocBlock are not reported

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#729]
Before (in PHP_CodeSniffer 2.0) some sniff from Core used by https://github.com/aik099/CodingStandard standard reported missing `@throws` tags in method DocBlock, when `throw new` were found in method body.

This stopped working at some point.

Example method:

```php
/**
 * Returns scoped config setting name.
 *
 * @param integer $scope Scope.
 *
 * @return string
 */
private function _getScopedName($scope)
{
	if ( $scope === self::SCOPE_GLOBAL ) {
		return 'global-settings.' . $this->_name;
	}

	if ( !$this->_workingCopyUrl ) {
		throw new \LogicException(
			'Please call setWorkingCopyUrl() prior to calling ' . __METHOD__ . '() method.'
		);
	}

	$wc_hash = substr(hash_hmac('sha1', $this->_workingCopyUrl, 'svn-buddy'), 0, 8);

	return 'path-settings.' . $wc_hash . '.' . $this->_name;
}
```

**Comment by gsherwood** [comment]
Running PHPCS 2.0.0 over this code gives the same results as running the latest PHPCS version. They both throw a Squiz.Commenting.FunctionCommentThrowTag.Missing error.

Do you have the specific error message and code that isn't being thrown?

**Comment by aik099** [comment]
It turned out that I was seeing error from PhpStorm inspection (see https://youtrack.jetbrains.com/issue/WI-29151) and `Squiz.Commenting.FunctionCommentThrowTag` was never enabled in my standard.

Thanks for the intel about that sniff name.
